sw/cheri/cheriot_toolchain.cmake

```diff
@@ -23,13 +23,13 @@ string(CONCAT CMAKE_CXX_FLAGS_INIT
   "-std=c++20 -mcpu=cheriot -mabi=cheriot -mxcheri-rvc -mrelax -fshort-wchar"
   " -Os -g -fomit-frame-pointer -fno-builtin -fno-exceptions"
   " -fno-asynchronous-unwind-tables -fno-c++-static-destructors -fno-rtti"
-  " -Wall -Werror -fvisibility=hidden -fvisibility-inlines-hidden"
+  " -Wall -fvisibility=hidden -fvisibility-inlines-hidden"
   " -DDEVICE_EXISTS_uart"
 )
 
 string(CONCAT CMAKE_C_FLAGS_INIT
   "-mcpu=cheriot -mabi=cheriot -mxcheri-rvc -mrelax -fshort-wchar"
-  " -Os -g -fomit-frame-pointer -fno-builtin -Wall -Werror"
+  " -Os -g -fomit-frame-pointer -fno-builtin -Wall"
```

Comment:
Why are you removing the `-Werror` here?

Reply:
See the commit message of the first commit

Reply:
Oh do you mean just remove it from CXX flags but not C flags?